### PR TITLE
PUT /settings requires a correct version field.

### LINKED ISSUE #4040: Show Mute is not working in Diagnostics
### Actual Behavior

Mute toggle is hiding warnings as expected. However, if we hit the 'Show Muted' toggle, the muted warnings do not show up. 

### Steps to Reproduce

1. Fresh Install Rancher Desktop/ Start Rancher Desktop after a Factory reset
2. Set path management strategy to Manual
3. Go to Diagnostics tab from the main window
4. Mute a warning message
5. Hit 'Show Muted' button

### Result

 Once we hit the 'Show Muted' toggle, the muted warnings do not show up. 
show muted diagnostics

### Expected Behavior

Once we select Show Muted toggle, all muted warnings must be visible in the Diagnostics tab.

### Additional Information

_No response_

### Rancher Desktop Version

1.7.0-419-gbf3825e1

### Rancher Desktop K8s Version

1.25.6

### Which container engine are you using?

containerd (nerdctl)

### What operating system are you using?

macOS

### Operating System / Build Version

macOS Monterey

### What CPU architecture are you using?

arm64 (Apple Silicon)

### Linux only: what package format did you use to install Rancher Desktop?

None

### Windows User Only

_No response_

### PR DESCRIPTION
**Pull request by ericpromislow**
Fixes #4040

### CODE REVIEW

**Review by jandubois — APPROVED** [review]
Thanks, LGTM, but I want to test the CI build before merging.

**Review by rak-phillip — APPROVED** [review]
LGTM as well, ready to merge after after CI passes

**Pull request merged**
